staging/src/k8s.io/pod-security-admission/admission/api/load/load_test.go

```diff
@@ -121,6 +121,29 @@ func TestLoadFromFile(t *testing.T) {
 		}
 	}
 
+	// valid file
```

Comment:
nit: prefer
```suggestion
	// valid v1 file
```
(and v1beta1, above)